.tekton/integration/validate-migration.sh

```diff
@@ -0,0 +1,458 @@
+#!/usr/bin/env bash
+
+# Validate migration file introduced by a branch.
+#
+# This script can be run in the CI against a PR or in local from a topic branch.
+# Before run, all local changes have to be committed.
+#
+# Network is required to execute this script.
+#
+# Checks are implemented as functions whose name has prefix `check_`. Each of
+# them exits with status code 0 to indicate pass, otherwise exits
+# script execution immediately with non-zero code.
+#
+# Test with a local cluster:
+#
+# - Before starting test with local cluster, launch one by `kind create
+#   cluster'.  Then, install tekton pipelines
+#   https://tekton.dev/docs/pipelines/install/#installation
+#
+# - Set IN_CLUSTER to true.
+#
+# NOTE: file moved here for testing purposes, move to permanent home before closing STONEINTG-1235
```

Comment:
Should we merge this file or remove it before merging?

Reply:
I completely forgot about that. Who else is using the validate-migration script? I can move the task to the appropriate catalog

Reply:
Hi @hongweiliu17 you might know more about this ^^

Reply:
https://github.com/konflux-ci/konflux-test-tasks/blob/main/.github/workflows/check-task-migration.yaml#L34 is used by this github workflow. I think it can be removed if the new integration test works.